ios/Classes/BetterPlayerPlugin.m

```diff
@@ -148,15 +148,19 @@ - (void) setupRemoteCommands:(BetterPlayer*)player  {
     [commandCenter.playCommand addTargetWithHandler: ^MPRemoteCommandHandlerStatus(MPRemoteCommandEvent * _Nonnull event) {
         if (_notificationPlayer != [NSNull null]){
             _notificationPlayer.eventSink(@{@"event" : @"play"});
+            return MPRemoteCommandHandlerStatusSuccess;
+        } else {
+            return MPRemoteCommandHandlerStatusCommandFailed;
```

Comment:
Sorry.
Do you know how to reproduce for reaching this line?
(In other words, "how do we get `_notificationPlayer == [NSNull null]` when receiving `playCommand`?".)

Reply:
@noboru-i 
Before i found the problem of `_willStartPictureInPicture` flag, i already got the case that `_notificationPlayer == [NSNull null]`. But it's really hard to reproduce, even though the Player can't play very often so i think the problem is the other one and then i found that `_willStartPictureInPicture` flag.

Reply:
@mvn-hienvu-hn 
I have understood there is a problem with `_willStartPictureInPicture` flag.
After solving that problem, these line changes are required?
(I am afraid of other effects.)